packages/server/tests/acceptance/conformityTests.spec.ts

```diff
@@ -0,0 +1,271 @@
+};
```

Comment:
Maybe we should add contract deployments via eip 2930 and eip1559 as well?

Reply:
the tests here are replicated from the test in the execution apis repo
so i just execute them
we can create additional tests, but i think they should be a separate issue?

Reply:
Yeah additional test can be captured in a separate ticket.
It's unclear that we're replicating tests from the package.
Maybe some more comments would help

Reply:
https://github.com/hashgraph/hedera-json-rpc-relay/issues/1730
Created an issue for this. Also added some comments to clarify what we are doing here